drivers/lcd/memlcd.c

```diff
@@ -165,6 +182,7 @@ static int memlcd_setcontrast(struct lcd_dev_s *dev, unsigned int contrast);
  * Private Data
  ****************************************************************************/
 
+static uint8_t _sharpmem_vcom;
 static uint8_t g_runbuffer[MEMLCD_BPP * MEMLCD_XRES / 8];
```

Comment:
let's move the global variables into memlcd_dev_s

Reply:
g_runbuffer could not be moved to memlcd_dev_s, because it is used in planeinfo structure